modules/security-jwt/src/main/resources/application.properties

```diff
@@ -1,3 +1,20 @@
-#security.authentication.keystore.keystorelocation=C:/Devon-dist_3.0.0/jwtkeystore.p12
+## below properties need to be configured in config\application.properties
```

Comment:
do not provide `application.properties` as part of this module at all.
Add this to the application template and/or documentation instead.